buildscripts/kokoro/unix.sh

```diff
@@ -19,26 +25,14 @@ cd $(dirname $0)/../..
 # Proto deps
 export PROTOBUF_VERSION=3.5.1
 
-# TODO(zpencer): if linux builds use this script, then also repeat this process for 32bit (-m32)
-# Today, only macos uses this script and macos targets 64bit only
-
-CXX_FLAGS="-m64" LDFLAGS="" LD_LIBRARY_PATH="" buildscripts/make_dependencies.sh
-
-# the install dir is hardcoded in make_dependencies.sh
-PROTO_INSTALL_DIR="/tmp/protobuf-${PROTOBUF_VERSION}/$(uname -s)-$(uname -p)"
-
-if [[ ! -e /tmp/protobuf ]]; then
-  ln -s $PROTO_INSTALL_DIR /tmp/protobuf;
-fi
+# ARCH is 64 bit unless otherwise specified.
+export ARCH="${ARCH:-64}"
```

Comment:
Could we avoid exporting this? It can be hard to reverse-engineer everything that is looking at it.

```
ARCH="${ARCH:-64}"
ARCH="$ARCH" buildscripts/make_dependencies.sh
```

Reply:
Removed export.